### ISSUE TITLE
Incompatible mod set with latest version of voicechat & audioplayer

### ISSUE THREAD

**Issue by ComputerJock** [henkelmax/audio-player#26]
I was getting my 1.19.2 installation updated to the latest audio mods:
voicechat-fabric-1.19.2-2.3.26.jar
audioplayer-1.19.2-1.4.5.jar

I now get this message:

Incompatible mod set!
net.fabricmc.loader.impl.FormattedException: Mod resolution encountered an incompatible mod set!

A potential solution has been determined:
- Install voicechat, version 1.19.2-2.3.3 or later.
Unmet dependency listing:
- Mod 'AudioPlayer' (audioplayer) 1.19.2-1.4.5 requires version 1.19.2-2.3.3 or later of voicechat, which is missing!

Steps to reproduce the issue
Install the above two mods in a 1.19.2 fabric installation.

Expected behavior
No error message.

Log files
https://gist.github.com/ComputerJock/54ebbf52748b353d805447dea4f76322

Versions

Minecraft version 1.19.2
Fabric version
Mod/Plugin version: listed above
Other mods
List is in the log file above.

I removed all mods except for fabric-api and these two audio mods. Got the same error message.

Note: 
Reinstalling voicechat-fabric-1.19.2-2.3.18.jar eliminates the error message and the game starts successfully.

**Comment by henkelmax** [comment]
I just tested it with the latest version of both mods and it works perfectly fine.
Please check if this happens with a clean fabric install with just these two mods.

**Comment by ComputerJock** [comment]
You're right -- it works with a clean install & only the two audio mods & fabric-api:
fabric-api-0.68.0+1.19.2.jar
voicechat-fabric-1.19.2-2.3.26.jar
audioplayer-1.19.2-1.4.5.jar

I'm using MultiMC. Is there some sort of cache that could hold some sort of previous settings for the installed mods?

----

I just noticed that fabric-api has been updated to: fabric-api-0.72.0+1.19.2.jar

I went back to my 1.19.2 installation that had previously failed with the mod mismatch messages, installed this fabric-api:
voicechat-fabric-1.19.2-2.3.26.jar
audioplayer-1.19.2-1.4.5.jar
fabric-api-0.72.0+1.19.2.jar
...
and all the other mods I use.

Wouldn't you know? It now works. 

I don't know if it was some sort of user error on my part or what. Sorry for wasting your time.
